src/ngrest/plugins/Injector.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace luya\admin\ngrest\plugins;
+
+use luya\admin\ngrest\base\Plugin;
+use luya\helpers\Html as HtmlHelper;
+
+/**
+ * Create a text input select for a given field.
+ *
```

Comment:
@boehsermoe Could you please write a full example of what this does and how to use as php doc? Its hardly to assume what it does ;-) Directly in the php doc would be nice.